ee/tools/puppet/fleetdm/README.md

```diff
@@ -0,0 +1,98 @@
+# fleetdm
+
+## Table of Contents
+
+1. [Description](#description)
+1. [Setup - The basics of getting started with fleetdm](#setup)
+    * [Setup requirements](#setup-requirements)
+    * [Beginning with fleetdm](#beginning-with-fleetdm)
+1. [Usage - Configuration options and additional functionality](#usage)
+1. [Limitations - OS compatibility, etc.](#limitations)
+1. [Development - Guide for contributing to the module](#development)
+
+## Description
+
+Manage MDM settings for macOS devices using [Fleet](https://fleetdm.com)
+
+## Setup
+
+### Setup Requirements
+
+This module requires to add `fleetdm` as a reporter in your `report` settings,
+this helps Fleet understand when your Puppet run is finished and assign the
+device to a team with the necessary profiles.
+
+For example, in your server configuration:
+
+```
+reports = http,fleetdm
+```
+
+To communicate with the Fleet server, you also need to provide your server URL
+and a token as Hiera values:
+
+```yaml
+---
+fleetdm::host: https://example.com
+fleetdm::token: my_token 
+```
+
+Note: for the token, we recommend using an [API-only user][1]
```

Comment:
With sufficient role (the way it's looking is that it would require global admin or gitops? It needs to create teams if needed, and create/update MDM profiles).

Reply:
Good point, I'll add gitops for now, and adjust as the picture is more clear.